### PR TITLE
Add Analyze MA tool.

### PR DESCRIPTION
**Pull request by kberg**
image

### CODE REVIEW

**Comment by bafolts** [comment]
Love that this project is far enough along that a page like this makes sense to add.

**Comment by kberg** [comment]
It's not an additional page. It's a command-line tool.

**Comment by kberg** [comment]
Still, I'm happy to have it.

File: src/tools/analyze_ma.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
```
Comment (bafolts) [inline src/tools/analyze_ma.html:2]: I believe the `meta` element goes inside the `<head>` element

File: src/tools/analyze_ma.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<html>
+<head>
+
+<!-- Load d3.js -->
+<script src="https://d3js.org/d3.v4.js"></script>
+
```
Comment (bafolts) [inline src/tools/analyze_ma.html:8]: This needs a closing `</head>` and an opening `<body>` element.

File: src/tools/analyze_ma.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+<form>
+  <label for="runs">Runs: </label></label><input type="text" id="runs" value="10000" name="runs"></input><br/>
```
Comment (bafolts) [inline src/tools/analyze_ma.html:10]: Duplicate `</label>` here and on next 3 lines.

**Review by bafolts — APPROVED** [review]

**Comment by bafolts** [comment]
As it is a command line tool you may be able to generate the SVG and save it to a file versus using the HTML page. Taking in the arguments through CLI.

File: src/tools/analyze_ma.html (hunk 1 above)
Comment (kberg) [reply src/tools/analyze_ma.html:2]: Done.

File: src/tools/analyze_ma.html (hunk 2 above)
Comment (kberg) [reply src/tools/analyze_ma.html:8]: Done.

File: src/tools/analyze_ma.html (hunk 3 above)
Comment (kberg) [reply src/tools/analyze_ma.html:10]: Fixed. Thanks!

**Comment by kberg** [comment]
> As it is a command line tool you may be able to generate the SVG and save it to a file versus using the HTML page. Taking in the arguments through CLI.

Absolutely. Anyone who wants to take the next step now has something to bulid on.

**Pull request merged**